### PR TITLE
JBPM-7118: Stunner - Mediators (zoom,pan) not working properly (2)

### PR DESCRIPTION
**Pull request by karreiro**
See:
- https://github.com/ahome-it/lienzo-core/pull/218
- https://issues.jboss.org/browse/JBPM-7118

---

No feature is being added by this PR, and no behavior is being broken. I'm just syncing the PR with the blessed repository.

Do not merge before getting the https://github.com/ahome-it/lienzo-core/pull/218 merged.

---

Part of an ensemble:
- https://github.com/kiegroup/lienzo-core/pull/26
- https://github.com/kiegroup/lienzo-tests/pull/20

### CODE REVIEW

**Comment by romartin** [comment]
hey @karreiro , but should we merge it?

**Comment by karreiro** [comment]
@romartin I think that that it's a good idea to wait for the https://github.com/ahome-it/lienzo-core/pull/218 get merged, and only then to merge this one.

**Comment by karreiro** [comment]
@romartin Since the https://github.com/ahome-it/lienzo-core/pull/218 is merged, **we can marge this one and the https://github.com/kiegroup/lienzo-tests/pull/20**.

I know that the https://github.com/ahome-it/lienzo-tests/pull/22 is not merged yet. But having the https://github.com/ahome-it/lienzo-core/pull/218 merged it's good enough to get our PRs (kiegroup ones) merged.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by romartin** [comment]
Hey @karreiro , sorry for being late on this!!! we're absolutely busy with some other blocker issues :( 
Anyway I was going to merge this but I'm a bit lost now sorry... please try to clarify this:
* I don't understand why you said this PR should be merged after https://github.com/ahome-it/lienzo-core/pull/218, as we don't depend on this at all right now.. I mean, there should be no dependency to the ahome-it:lienzo-core, we're just forking here... 
* Finally are these PR the ones to be merged for this, or am I missing something? IIRC there exist:
** https://github.com/kiegroup/lienzo-core/pull/26
** https://github.com/kiegroup/lienzo-tests/pull/20
** https://github.com/ahome-it/lienzo-tests/pull/22

Is that correct?
Again - sorry for being a bit later on this review, once you clarify this I'll be happy to merge :)
Thanks!!

**Comment by karreiro** [comment]
Hi @romartin, no problem at all! Sorry for being late too, I was on PTO.

You're right, we don't depend on the blessed repository. However, @mdproctor was doing an effort to keep our Lienzo fork synced with the blessed.

I talked to Mark on IRC and we noticed that the JBPM-7118 was not fixed in the blessed repo and that we could incorporate part of [Daniel's ideas](https://github.com/ahome-it/lienzo-core/issues/211#issuecomment-381811416) in the [PR I opened in the blessed](https://github.com/ahome-it/lienzo-core/pull/218).

In order to keep both repositories synced, I opened this PR. This one does not change any behavior, we're just keeping our code synced with the blessed. Notice that we're not changing any asserts [here]( https://github.com/kiegroup/lienzo-tests/pull/20).

In summary, we just need to merge [this PR](https://github.com/kiegroup/lienzo-core/pull/26) and the https://github.com/kiegroup/lienzo-tests/pull/20. :-)

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by romartin** [comment]
hey @hasys I need a QE review for this, thanks!

**Comment by karreiro** [comment]
Jenkins execute full downstream build.

**Comment by karreiro** [comment]
Jenkins execute full downstream build.
